src/displayapp/screens/Dice.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "displayapp/screens/Screen.h"
+#include "displayapp/widgets/Counter.h"
+#include "components/datetime/DateTimeController.h"
+#include <lvgl/lvgl.h>
+
+namespace Pinetime::Applications::Screens {
+  class Dice : public Screen {
+    public:
+      Dice(DisplayApp* app, Controllers::DateTime& dateTime);
+      ~Dice() override;
+      void Roll();
+      void NextColor();
+
+    private:
+      Controllers::DateTime& dateTime;
```

Comment:
because you need the dateTime object only in the constructor you can remove the stored reference here

Reply:
addressed in 6c7d8c2fcae8f649b94a3d23fe4b42c4ea1fd806